torch/csrc/jit/codegen/cuda/fusion.h

```diff
@@ -229,6 +229,28 @@ class TORCH_CUDA_CU_API Fusion final {
   std::unordered_set<int> getOutputAliasIndices() const;
   std::vector<std::pair<int, int>> getInputAliasIndices() const;
 
+  // mark input at index to be in channels last format
+  void setChannelsLastInputIndices(int index) {
```

Comment:
I talked to you about this live.  The naming here seems confusing to me where you are marking an Input as Channels last but the mechanism by which you do so is recording the index of the specific input.  Perhaps it would be less confusing if you just named the method `setChannelsLastInput()` and remove the fact that you specifying the input via an index from the name.